kedro-org/kedro-viz
# Test Flowchart as subpackage

## Description

Steps to test flowchart with kedro-layout project

- do npm install 
- in the nodule_modules you should see @quantumblack/kedro-viz-flowchart folder 
- go to your kedro-layout project and do npm install ../path/to/the/kedro-viz/node-modules/@quantumblack/kedro-viz-flowchart or 
- npm install ../path/to/kedro-viz/lib/utils/graph
- and then change your import to 'import { graphNew } from '@quantumblack/kedro-viz-flowchart' 
- npm start 

## Development notes



## QA notes



## Checklist
